mindmeld/components/_elasticsearch_helpers.py

```diff
@@ -15,14 +15,14 @@
 import logging
 import os
 
-from elasticsearch5 import ConnectionError as EsConnectionError
-from elasticsearch5 import (
+from elasticsearch import ConnectionError as EsConnectionError
```

Comment:
Is the `elasticsearch ` lib for ES 7 or does it work across all ES versions?

Reply:
It works across all versions, but you need to have your python client match versions with your ES client: https://elasticsearch-py.readthedocs.io/en/master/ 

I think we should recommend that everyone installs ES7 and the matching client in the documentation and requirements, but need to think more about backwards compatibility.

Reply:
Verified that it works across all versions for the methods we are using